tests/baselines/reference/extractMethod1.js

```diff
@@ -0,0 +1,98 @@
+==ORIGINAL==
+namespace A {
+    let x = 1;
+    function foo() {
+    }
+    namespace B {
+        function a() {
+            let a = 1;
+        
+            let y = 5;
+            let z = x;
+            a = y;
+            foo();
+        }
+    }
+}
+==SCOPE::function a==
+namespace A {
+    let x = 1;
+    function foo() {
+    }
+    namespace B {
+        function a() {
+            let a = 1;
+        
+            newFunction();
+
+            function newFunction() {
+                let y = 5;
+                let z = x;
+                a = y;
+                foo();
+            }
+        }
+    }
+}
+==SCOPE::namespace B==
+namespace A {
+    let x = 1;
+    function foo() {
+    }
+    namespace B {
+        function a() {
+            let a = 1;
+        
+            ({ a } = newFunction(a));
+        }
+
+        function newFunction(a: any) {
```

Comment:
Something's going weird with the unit test harness setup - this gets `: number` when I do it in the editor